packages/shopping-cart/src/use-refetch-on-focus.ts

```diff
@@ -44,7 +54,8 @@ export default function useRefetchOnFocus(
 		function isOffline(): boolean {
 			try {
 				return ! window.navigator.onLine;
-			} catch {
+			} catch ( err ) {
+				debug( 'failed to check onLine status with error', err );
```

Comment:
I'm not sure, actually, but as accessing `window` always seems risky (eg: with server side rendering), and because this behavior is non-essential, I wanted a silent fallback. You're right, though, that if that does happen maybe it's safer to treat this check as not conclusive, and therefore return false.